profile.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Is this intentional? This script is nowhere near usable by anyone who doesn't like pain :)

Reply:
No idea what this is!

Reply:
Somehow got pulled in with s3-related stuff.  Can delete.